### PR TITLE
Tweak: add refund reason and Invoice number titles to the document abstract class

### PR DESCRIPTION
**Pull request by alexmigf**

### CODE REVIEW

**Comment by MohamadNateqi** [comment]
@alexmigf I was wondering if it is OK to add these methods to this plugin for strings introduced in the Pro version.
I was wondering if it's better to extend the `Order_Document_Methods` class and define these methods there.
What do you think?

**Comment by alexmigf** [comment]
@MohamadNateqi I have moved them to the methods abstract, which is extended by the document abstract.

**Comment by MohamadNateqi** [comment]
@alexmigf Since there are now many methods related to the get and print strings used in the templates, maybe we can introduce a trait and move them all there.

What do you think?

**Comment by alexmigf** [comment]
> Since there are now many methods related to the get and print strings used in the templates, maybe we can introduce a trait and move them all there.

I don't see the benefit, as these methods are only used by document classes. Perhaps we could reconsider how to handle this more effectively after merging the PSR-4 PR.

File: includes/documents/abstract-wcpdf-order-document-methods.php [hunk 1]
```diff
@@ -1313,6 +1313,47 @@ public function get_display_date_label( $date_string ) {
...
+	}
+	
+	/**
+	 * Display the title for the refund reason
+	 * 
+	 * @return void
+	 */
+	public function refund_reason_title(): void {
+		echo $this->get_refund_reason_title();
+	}
```
[the remark this answers is not in the document]
Comment (MohamadNateqi) [reply includes/documents/abstract-wcpdf-order-document-methods.php:1356]: Understood now.

**Review by MohamadNateqi — APPROVED** [review]
LGTM!

**Comment by MohamadNateqi** [comment]
> I don't see the benefit, as these methods are only used by document classes.

The only reason I mentioned this was to gather them in another class for a cleaner implementation; otherwise, as you said, there could not be any other benefits.

**Pull request merged**